Decent.Minecraft.Client/RailType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Decent.Minecraft.Client
+{
+    /// <summary>
+    /// The type of rail.
+    /// </summary>
+    public enum RailType : byte
```

Comment:
Its the same as all the other Enums we have to directions, etc. Its not Java specific and needs to be public for people to call the API's.